scripts/docs_template.txt

```diff
@@ -0,0 +1,9 @@
+*lspconfig-all*                              *lspconfig-server-configurations*
+
+LSP configurations provided by nvim-lspconfig are listed below.
+
```

Comment:
The `Type |gO| to see the table of contents.` should be included, same as all builtin help files. `gO` is not as well-known as `K` or `<c-]>`.